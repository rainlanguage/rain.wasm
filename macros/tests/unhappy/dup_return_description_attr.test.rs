#[macro_use]
extern crate wasm_bindgen_utils_macros;

struct TestStruct;

#[wasm_export]
impl TestStruct {
    #[wasm_export(return_description = "first description", return_description = "second description")]
    pub async fn some_method_with_dup_return_desc(arg: String) -> Result<String, Error> {
        Ok(String::new())
    }
}

fn main() {}