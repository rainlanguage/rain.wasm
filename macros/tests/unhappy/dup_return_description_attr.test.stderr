error: duplicate `return_description` attribute
 --> tests/unhappy/dup_return_description_attr.test.rs:8:61
  |
8 |     #[wasm_export(return_description = "first description", return_description = "second description")]
  |                                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
